hazelcast-vnet/README.md

```diff
@@ -0,0 +1,14 @@
+# Deploy a Hazelcast Cluster
```

Comment:
What's the significance of the "vnet" in the folder name? How about hazelcast-cluster?